weather-app/src/app/shared/components/current-weather/current-weather.component.html

```diff
@@ -5,7 +5,13 @@
             February 7th, 2022
         </p>
     </div>
-    <a href='' class="current-weather__city">Kyiv, Ukraine</a>
+    <a                 
+        routerLink=":city/details"  
```

Comment:
remove this

Reply:
![image](https://user-images.githubusercontent.com/70651413/154222494-5c978a6e-6b9e-4962-b327-a56a68d2a922.png)